### PR TITLE
feat(docker-local): e2e linking and transfer with pisp demo

### PR DESCRIPTION
**Pull request by lewisdaly**
part of mojaloop/project#2263

I'll update the readme with instructions for how to run e2e and linking and transfers with the pineapplepay demo in a followup PR

### CODE REVIEW

File: docker-local/auth-service/spec_files/rules_response/default.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
+[
```
Comment (kleyow) [inline docker-local/auth-service/spec_files/rules_response/default.json:1]: nit: this should be removed

**Review by kleyow — APPROVED** [review]

**Review by kleyow — APPROVED** [review]

**Comment by lewisdaly** [comment]
@kleyow I bet you circleci will now die for the next couple hours ;)

**Comment by kleyow** [comment]
I've been wondering why branch builds pass more often than master after changes are merged in.

Been trying to lookup circleci memory management if there is any so we can at least stem the problem.

**Comment by lewisdaly** [comment]
I think it may have to do with what time we run CircleCI at... I noticed during my day, it seems to be fine

**Comment by lewisdaly** [comment]
@kleyow maybe we can try spinning up the containers 50% at a time.

**Pull request merged**